src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -387,6 +399,11 @@ export const Checkout = ({
         return
       }
     } catch (e) {
+      console.log(e.graphQLErrors, e.networkError)
+      if (isManualReviewRequired((e?.graphQLErrors || []) as GraphQLError[])) {
```

Comment:
hmm just interesting - when is error `undefined`?

Reply:
Never really, it's just that error was not typed. The question mark is not needed